### PR TITLE
Multiline output parameters and return statements

### PR DESCRIPTION
**Pull request by OTTTO**

### CODE REVIEW

File: docs/style-guide.rst [hunk 1]
```diff
@@ -650,6 +650,50 @@ No::
         doSomething();
     }
 
+Multiline output parameters and return statements should follow the same style recommended for `wrapping long lines <http://solidity.readthedocs.io/en/develop/style-guide.html#maximum-line-length>`_.
```
Comment (axic) [inline docs/style-guide.rst]: You should use internal links here. I think they have a format like `:ref:maximum-line-length`, but you have to check.

Comment (OTTTO) [reply docs/style-guide.rst]: done ✓

**Comment by chriseth** [comment]
Is the link correct now?

File: docs/style-guide.rst [hunk 2]
```diff
@@ -650,6 +650,50 @@ No::
         doSomething();
     }
 
+Multiline output parameters and return statements should follow the same style recommended for wrapping long lines found in the `Maximum Line Length`_ section.
```
Comment (axic) [inline docs/style-guide.rst]: Have you tried this way of referencing?

Other parts of the documentation use a different method:

The links have a format of 
```
:ref:`Maximum Line Length <maximum_line_length>`
```

but a reference must also be added to the original heading as
```
.. _maximum_line_length:
```

Comment (OTTTO) [reply docs/style-guide.rst]: If you look at the file right now, that's what I just tried and I can't get it to work

File: docs/style-guide.rst [hunk 3]
```diff
@@ -112,7 +112,7 @@ No::
         }
     }
 
-Maximum Line Length
+Maximum Line Length _maximum_line_length:
```
Comment (axic) [inline docs/style-guide.rst]: Has to be above the title and needs `..` prefix

Comment (OTTTO) [reply docs/style-guide.rst]: header is working now, `:ref:` is still failing

File: docs/style-guide.rst [hunk 4]
```diff
@@ -112,7 +112,8 @@ No::
         }
     }
 
-Maximum Line Length
+.. _maximum_line_length:
```
Comment (axic) [inline docs/style-guide.rst:115]: Not sure if that is the reason, but in other parts we use dashes and not underscores. That seems to be working `.. _maximum-line-length` and the reference must be changed too.

Also please keep an empty new line after it.

Comment (OTTTO) [reply docs/style-guide.rst:115]: I added the newline; but, in [the file I just checked](https://github.com/OTTTO/solidity/edit/develop/docs/security-considerations.rst) in the style guide I saw underlines.  
Also I noticed that in the GitHub preview, :ref:'words <reference>' [is always present](https://github.com/OTTTO/solidity/blob/develop/docs/security-considerations.rst); but, on [the official docs](http://solidity.readthedocs.io/en/develop/security-considerations.html) it becomes an internal link
So, if you agree with the changes we should be able to merge.

Comment (axic) [reply docs/style-guide.rst:115]: I personally don't mind which as long as the documentation compiles properly. Have you checked it? Just run `make html` in the `docs` directory and load it in a browser.

Comment (OTTTO) [reply docs/style-guide.rst:115]: it works 👍

**Pull request merged**